### PR TITLE
UG: Add change command and update remarks

### PR DESCRIPTION
**Pull request by Avalionnet**
- Add change command implementation details in the user guide.
- Update remark command usage details.

With this update, users will now be able to learn the syntax for rescheduling and modifying appointments.

### CODE REVIEW

**Review by YangYue128-helen — APPROVED** [review]
LGTM！

File: docs/UserGuide.md [hunk 1]
```diff
@@ -145,15 +145,34 @@ Examples:
...
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+We implemented our Remarks feature with the hopes of empowering you with the ability to add extra information to a patient's bio data! 
+So don't be shy and feel free to add anything under the sun that you feel apt for the patient. :smile:
+</div>
+<br> 
+![result for 'Add remark'](images/addRemark.jpg)
+<br>
+* Creates and **adds a remark** for the patient at the specified `INDEX`. The index here refers to the index number shown on the left side of the displayed patient list. 
+* Please note that the index used **must be a positive integer** 1, 2, 3, …​
+* A remark serves as an **optional field** for adding extra info to a patient's bio data and can be left blank if it is not applicable.  modified using the edit/remark command after it has been created.
```
Comment (JinHao-L) [inline docs/UserGuide.md]: I think the second line can be omitted. (Addressed in the 5th point)

**Review by JinHao-L — COMMENTED** [review]
LGTM! Just have to handle the merge conflict 👍

File: docs/UserGuide.md (hunk 1 above)
Comment (Avalionnet) [reply docs/UserGuide.md]: oh, thanks for spotting this bro 👍🏻 missed it out when I was editing the code haha.

Comment (Avalionnet) [reply docs/UserGuide.md]: oh, thanks for spotting this bro 👍🏻 missed it out when I was editing the code haha.

**Pull request merged**